docs/source/references/config.md

```diff
@@ -561,11 +582,11 @@ Has type `MetadataLimits`. Can be configured via environment variable `WSV_ASSET
 }
 ```
 
-### `wsv_configuration.length_limits`
+### `wsv_configuration.ident_length_limits`
 
-[`LengthLimits`] of identifiers in bytes that can be stored in the WSV.
+[`LengthLimits`] in bytes of identifiers that can be stored in the WSV.
```

Comment:
Hmm, i guess we can change `.len()` to `.chars().count()`. It would suit better

Reply:
The idea was to limit the size. But if we have chars count - size can be different each time depending on the language.

Reply:
Ikr, but it is weird in terms of ui. Users think about characters, not byte length

Reply:
Later account names will no longer be identifiers - and will just be data - if you remember from discussion about account id with Makoto-san. So I think it doesn't really matter in that case.

Reply:
It will be breaking change anyways (change from string to raw bytes), so i dont see a reason for limiting in bytes, instead of characters.

Reply:
Fixed